### ISSUE TITLE
Editor Tool for traversing assets and creating lists for script export

### ISSUE THREAD

**Issue by Inveracity** [beardfist/quadropus#100]

**Comment by lemonsaurus** [comment]
yeah I really wanted to do this but I'm not entirely sure it's feasible. I've looked into it and the only way I can find that makes sense is to create a special type of resource for a voice or a stem or whatever the thing is (like a new Godot Resource), and then doing an export that hints that resource type.

This would have the desired effect, but would require more setup when adding a new resource.
